RocketChat/Rocket.Chat
# [FIX] Navigate to home after deleting a channel from edit screen

Improved channel/team deletion Feedback



https://user-images.githubusercontent.com/23723464/115000823-c15dee00-9ec0-11eb-84b8-f84d72d352b6.mov






## Proposed changes (including videos or screenshots)





## Issue(s)
fixes: #21601 

## Steps to test or reproduce


Go to a channel info
Click on edit
Click delete
Now you can see the error. Channel deletes successfully. But it won't navigate back to home.


## Further comments
